Tests/Form/Type/DoctrineORMSerializationTypeTest.php

```diff
@@ -63,7 +64,12 @@ public function testBuildForm()
         $metadataFactory = $this->getMetadataFactoryMock();
         $registry = $this->getRegistryMock();
 
-        $type = new DoctrineORMSerializationType($metadataFactory, $registry, 'form_type_test', $this->class, 'serialization_api_write');
+        if (!method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
```

Comment:
For clearer logic, I would check if method_exists first. If not, other statement.